tests/functional/conftest.py

```diff
@@ -46,14 +46,23 @@ def login_as_user(live_server, context, user_dict):
     sets the session cookie.
     """
     user = factories.create_airlock_user(**user_dict)
-    session_store = Session.get_session_store_class()()  # type: ignore
-    session_store["user"] = user.to_dict()
-    session_store.save()
+
+    from django.contrib import auth
+    from django.test import RequestFactory
```

Comment:
do these imports need to be inside the function?

Reply:
The fix for this is now part of #837